auth/src/auth.cc

```diff
@@ -373,5 +373,18 @@ AUTH_RESULT_FN(Auth, SignInWithEmailAndPassword, AuthResult)
 
 AUTH_RESULT_FN(Auth, CreateUserWithEmailAndPassword, AuthResult)
 
+// Platform-specific implementation of UseUserAccessGroup.
```

Comment:
Don't pu tthe platform-specific implementation here, put it in auth_android and auth_desktop and auth_ios.